pkg/term/windows/api.go

```diff
@@ -0,0 +1,343 @@
+// +build windows
+
+package windows
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+
+	// "github.com/Sirupsen/logrus"
+)
+
+//===========================================================================================================
+// IMPORTANT NOTE:
+//
+//	The methods below make extensive use of the "unsafe" package to obtain the required pointers.
+//	Beginning in Go 1.3, the garbage collector may release local variables (e.g., incoming arguments, stack
+//	variables) the pointers reference *before* the API completes.
```

Comment:
Rethought this entire area.

-- I removed the *use* scheme entirely.
-- Integer values and pointers are safe since they will "live" entirely on the stack passed to Windows. Go can free whatever it desires without impinging on the values Windows will use.
-- I've updated all API methods to take pointers to all other values (e.g., structs). This should ensure that the Go GC will leave the backing objects alone.

I've updated the comments to reflect the new strategy.